src/main/java/net/hypixel/nerdbot/api/database/greenlit/GreenlitMessage.java

```diff
@@ -26,6 +27,7 @@ public class GreenlitMessage {
     private ObjectId id;
     private String userId, messageId, greenlitMessageId, suggestionTitle, suggestionContent, suggestionUrl, channelGroupName;
     private List<String> tags;
+    private List<String> positiveVoterIDs;
```

Comment:
To keep it uniform with other variable names I think it should be changed to `positiveVoterIds`